presentation/src/main/java/com/maxrzhe/presentation/ui/base/CoreFragment.kt

```diff
@@ -0,0 +1,24 @@
+package com.maxrzhe.presentation.ui.base
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+
+abstract class CoreFragment : Fragment() {
+
+    protected open fun initView() = Unit
+    protected open fun layoutId(): Int = 0
```

Comment:
change `open` to `abstract` as we need to use `layoutId` property for all classes-inheritor

Reply:
fixed